randovania/cli/commands/permalink.py

```diff
@@ -54,14 +75,15 @@ def permalink_command(args):
 
 
 def add_permalink_arguments(parser: ArgumentParser):
+    parser.add_argument("--seed-number", type=int, help="The seed number. Defaults to 0.")
     parser.add_argument(
-        "--game",
-        choices=[game.value for game in RandovaniaGame],
-        required=True,
-        help="The name of the game of the preset to use.",
+        "--preset-name",
+        type=str,
+        help="The name of the presets to use. Should be named <game>/<name>",
+        nargs="*",
+        default=[],
     )
-    parser.add_argument("--seed-number", type=int, help="The seed number. Defaults to 0.")
-    parser.add_argument("--preset-name", required=True, type=str, help="The name of the presets to use", nargs="+")
+    parser.add_argument("--preset-file", type=Path, help="The path of preset file to use", nargs="*", default=[])
```

Comment:
The help text should also use the plural because it accepts multiple files like e.g. `preset-name` accepts multiple names.
...which makes me wonder if it is also worth using the plural in both argument names itself.

Reply:
It's actually an argument you need to specify multiple times. Will change the name to be shorter and mention in the help.